source/client/options_impl.cc

```diff
@@ -644,6 +654,18 @@ OptionsImpl::OptionsImpl(int argc, const char* const* argv) {
       throw MalformedArgvException(e.what());
     }
   }
+  if (!user_defined_output_plugin_configs.getValue().empty()) {
+    for (const std::string& plugin_config_string : user_defined_output_plugin_configs.getValue()) {
```

Comment:
Can we add unit tests (or TODOs to add them later) that:

- verify the behavior where the user supplied config does parse, but does not validate.
- verify the behavior where multiple configs are passed by the user.

Reply:
Added a test for the validation failure. Added a TODO for multiple plugins, which isn't possible until we have a second kind of User Defined Output Plugin.